wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/unterbrechungsuhrzeit/UnterbrechungsUhrzeitModelTest.java

```diff
@@ -0,0 +1,22 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.service.unterbrechungsuhrzeit;
+
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class UnterbrechungsUhrzeitModelTest {
+
+    @Nested
+    class ConstructionTests {
```

Comment:
Den Test können wir uns aus meiner Sicht sparen da dies das Standardverhalten ist. In Verbindung mit dem zu löschen Konstruktor wird der Test aus obsolet.

Reply:
resolved with b23f842e7149e911d60b97ee4db95e097908e3fb